src/Components/LoginForm.tsx

```diff
@@ -0,0 +1,228 @@
+import React, {
+  ChangeEvent,
+  FormEvent,
+  useContext,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { AppContext } from '../AppContext';
+import { findUser, addUsers } from '../api/todos';
+import { Notification } from './Notification';
+
+const addPrevUser = (prevEmail: string) => {
+  const getPrevUsers: string[] | 0 = JSON
```

Comment:
not fixed

<img width="496" alt="Screenshot 2023-04-08 at 20 34 13" src="https://user-images.githubusercontent.com/32853034/230737444-81c9080e-3b40-4217-976c-c71dd9e4fa8e.png">

```suggestion
  const prevUsers: string[] | 0 = JSON
```